pkgs/os-specific/linux/oddjob/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, pkgs, stdenv, libxml2, dbus, pam, systemd, pkg-config, autoreconfHook }:
+stdenv.mkDerivation rec {
+
+  pname = "oddjob";
+  version = "0.34.7";
+
+  src = pkgs.fetchurl {
+     url =
+       "https://pagure.io/oddjob/archive/${pname}-${version}/oddjob-${pname}-${version}.tar.gz";
+     sha256 = "sha256-SUOsMH55HtEsk5rX0CXK0apDObTj738FGOaL5xZRnIM=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  buildInputs =
+    [ libxml2 dbus pam systemd ];
+
+  postPatch = ''
+    substituteInPlace configure.ac --replace 'SYSTEMDSYSTEMUNITDIR=`pkg-config --variable=systemdsystemunitdir systemd 2> /dev/null`' "SYSTEMDSYSTEMUNITDIR=${placeholder "out"}"
+    substituteInPlace configure.ac --replace 'SYSTEMDSYSTEMUNITDIR=`pkg-config --variable=systemdsystemunitdir systemd`' "SYSTEMDSYSTEMUNITDIR=${placeholder "out"}"
+  '';
+
+  configureFlags = [
+    "--prefix=${placeholder "out"}"
+    "--sysconfdir=${placeholder "out"}/etc"
+    "--with-selinux-acls=no"
+    "--with-selinux-labels=no"
+  ];
+
+  postConfigure = ''
+    substituteInPlace src/oddjobd.c --replace "globals.selinux_enabled" "FALSE"
+  '';
+
+  meta = with lib; {
+    description = "Odd Job Daemon";
+    homepage = "https://pagure.io/oddjob";
+    changelog = "https://pagure.io/oddjob/blob/master/f/ChangeLog";
```

Comment:
Could this be fixed to the branch of the version packaged instead?

Reply:
The changelog link?

Reply:
Yes.